util/crc32c_ppc.c

```diff
@@ -0,0 +1,101 @@
+//  Copyright (c) 2017 International Business Machines Corp.
+//  All rights reserved.
+//  This source code is licensed under the BSD-style license found in the
+//  LICENSE file in the root directory of this source tree. An additional grant
+//  of patent rights can be found in the PATENTS file in the same directory.
+//  This source code is also licensed under the GPLv2 license found in the
+//  COPYING file in the root directory of this source tree.
```

Comment:
I am still waiting to hear back from FB open-source legal team on the question of whether this kind of copyright header is acceptable.